app/controllers/graphql_controller.rb

```diff
@@ -30,4 +30,31 @@ def ensure_hash(ambiguous_param)
       raise ArgumentError, "Unexpected parameter: #{ambiguous_param}"
     end
   end
+
+  def current_user
+    if doorkeeper_token && !doorkeeper_token.expired?
+      return User.find(doorkeeper_token.resource_owner_id)
+    end
+
+    raise Doorkeeper::Errors::DoorkeeperError, "Invalid access token"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/d439434a286bd76fead4ad367fe89e1a1cb8da1b/app/controllers/graphql_controller.rb#L39"></span>Prefer single-quoted strings when you don't need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>